src/components/Configure/content/ConfigureInstallationBase.tsx

```diff
@@ -29,17 +31,24 @@ export function ConfigureInstallationBase(
     onSave, onReset, isLoading, isCreateMode = false,
   }: ConfigureInstallationBaseProps,
 ) {
+  const { installation } = useInstallIntegrationProps();
   const { hydratedRevision, loading } = useHydratedRevision();
   const { configureState, selectedObjectName } = useSelectedConfigureState();
 
+  // check if selected object is completed.
```

Comment:
Can you add that, in the case the object does not have any configuration required, it is also considered "complete"?

Reply:
The code doesn't check for required fields. The code checks if the object exists in the config. If the object doesn't exists, we assume it is not completed by the user yet, therefore validation is not enforced.